project/frontend/src/components/ImageSlider.js

```diff
@@ -0,0 +1,186 @@
+import './ImageSlider.css';
+
+import React, { useState, useReducer } from 'react';
+import { useSwipeable } from 'react-swipeable';
+
+import Next from '../assets/next.svg';
+import Prev from '../assets/prev.svg';
+
+/**
+ * Types of swipe actions.
+ * @enum {number}
+ */
+const Action = {
+  SWIPING: 0,
+  SWIPED_LEFT: 1,
+  SWIPED_RIGHT: 2,
+  PREV_SLIDE: 3,
+  NEXT_SLIDE: 4,
+};
+
+/**
+ * The default width of a slide.
+ *
+ * @type {number}
+ */
+const width = 344;
```

Comment:
Does this scale with the width of the screen?

Reply:
No, right now the card dimensions are hard-coded. I do think it is necessary to scale the card based on different screen sizes but I'd like to make those changes in a future commit, as scaling the card sizes will have a lot to do with the CardDeck and CardStack components as well.

Reply:
Ok, lets leave a TODO here to come back and do that!